sweagent/environment/swe_env.py

```diff
@@ -153,10 +160,11 @@ def _reset_repository(self) -> None:
             # todo: Currently has swe-ft specific change: The original repo.copy isn't called, because the repo is already
             # present. However, reset --hard <BRANCH> also doesn't work. So modified it here to do a checkout instead.
             startup_commands = [
-                f"cd /{self.repo.repo_name}",
+                f"cd {repo_base_dir}/{self.repo.repo_name}",
                 "export ROOT=$(pwd -P)",
                 "git status",
-                "git fetch",
+                # "git fetch", # it seems useless
+                "git reset --hard", # solve typechange
```

Comment:
We cannot do `git reset --hard`. Would break certain swe-bench instances, for example